src/toast/ops/pointing_detector/pointing_detector.py

```diff
@@ -226,6 +238,18 @@ def _exec(self, data, detectors=None, use_accel=None, **kwargs):
                 use_accel=use_accel,
             )
 
+        # Optionally apply HWP deflection
+        if self.hwp_deflection_radius is not None and \
+           self.hwp_deflection_radius.value != 0:
+            xaxis, yaxis, zaxis = np.eye(3)
+            deflection = qa.rotation(
+                xaxis, self.hwp_deflection_radius.to_value(u.radian)
+            )
```

Comment:
I think the total deflection should be the quaternion product of this rotation and the phase rotation about the `zaxis`.

Reply:
I believe this is the case